inspector.ipxe

```diff
@@ -3,6 +3,6 @@
 :retry_boot
 echo In inspector.ipxe
 imgfree
-kernel --timeout 60000 http://IRONIC_IP:HTTP_PORT/images/ironic-python-agent.kernel ipa-inspection-callback-url=http://IRONIC_IP:5050/v1/continue ipa-inspection-collectors=default,extra-hardware,logs systemd.journald.forward_to_console=yes BOOTIF=${mac} ipa-debug=1 ipa-inspection-dhcp-all-interfaces=1 ipa-collect-lldp=1 initrd=ironic-python-agent.initramfs || goto retry_boot
+kernel --timeout 60000 http://IRONIC_IP:HTTP_PORT/images/ironic-python-agent.kernel ipa-inspection-callback-url=http://IRONIC_IP:5050/v1/continue ipa-api-url=http://IRONIC_IP:6385 ipa-inspection-collectors=default,extra-hardware,logs systemd.journald.forward_to_console=yes BOOTIF=${mac} ipa-debug=1 ipa-inspection-dhcp-all-interfaces=1 ipa-collect-lldp=1 initrd=ironic-python-agent.initramfs || goto retry_boot
```

Comment:
You may want to set `console` here too if you want to see inspector logs (`pxe_append_params` doesn't apply to inspector).

Reply:
Hmm that's strange, I see them after adding the WIP commit that makes pxe_append_params only log to ttyS0

```
[    0.000000] Command line: deploy_kernel selinux=0 troubleshoot=0 text nofb nomodeset vga=normal console=ttyS0 ipa-debug=1 ipa-api-url=http://[fd00:1101::3]:6385 BOOTIF=00:f0:37:af:99:60 initrd=deploy_ramdisk
...
2020-07-01 08:06:16.359 1069 INFO ironic_python_agent.inspector [-] posting collected data to http://[fd00:1101::3]:5050/v1/continue^[[00m^M
2020-07-01 08:06:16.359 1069 DEBUG ironic_python_agent.inspector [-] collected data: {'inventory': {'interfaces': [<ironic_python_agent.hardware.NetworkInterface object at 0x7f143377e6a0>, <ironic_python_agent.hardware.NetworkInterface object at 0x7f1433775b70>], 'cpu': <ironic_python_agent.hardware.CPU object at 0x7f1433785ba8>, 'disks': [<ironic_python_agent.hardware.BlockDevice object at 0x7f1433785fd0>], 'memory': <ironic_python_agent.hardware.Memory object at 0x7f143379bcc0>, 'bmc_address': '0.0.0.0', 'bmc_v6address': '::/0', 'system_vendor': <ironic_python_agent.hardware.SystemVendorInfo object at 0x7f143379ef28>, 'boot': <ironic_python_agent.hardware.BootInfo object at 0x7f1439d84b70>, 'hostname': 'worker-1.ostest.test.metalkube.org'}, 'root_disk': <ironic_python_agent.hardware.BlockDevice object at 0x7f1433785fd0>, 'boot_interface': '00:f0:37:af:99:60', 'error': None} call_inspector /usr/lib/python3.6/site-packages/ironic_python_agent/inspector.py:120^[[00m^M
2020-07-01 08:06:16.775 1069 INFO ironic_python_agent.inspector [-] inspection finished successfully^[[00m^M
2020-07-01 08:06:16.776 1069 INFO ironic_python_agent.agent [-] No ipa-api-url configured, Heartbeat and lookup skipped for inspector.^[[00m^M
```

Is the `systemd.journald.forward_to_console=yes` perhaps already enabling `console` implicitly?